traq/tickets/views/__init__.py

```diff
@@ -18,6 +19,8 @@
 @permission_required('todos.add_todo')
 def detail(request, ticket_id):
     ticket = get_object_or_404(Ticket, pk=ticket_id)
+    #get all other tickets assigned to the user
+    tickets = Ticket.objects.filter(assigned_to=request.user).values_list('pk', flat=True)
```

Comment:
Is this a necessary var? You pass it to the template, but I don't see it used in there.